.github/workflows/rhub.yaml

```diff
@@ -0,0 +1,115 @@
+# This workflow is based on R-hub's generic GitHub Actions workflow file.
+# Its canonical location is at
+# https://github.com/r-hub/actions/blob/main/workflows/rhub.yaml
+
+name: R-hub 🌐
+
+on:
```

Comment:
add on schedule trigger OR: add to `.github/workflows/scheduled.yaml.shared`

Reply:
oh it looks like we are not using `scheduled.yaml.shared` in the repos ([example](https://github.com/insightsengineering/teal/blob/main/.github/workflows/scheduled.yaml)). Then it means that this has to be added into the repos

Reply:
Yes, currently we have `scheduled.yaml` and `on-demand.yaml` workflows in most repositories, and both of them have `scheduled` and `workflow_dispatch` triggers. We could add the R-hub workflow to one of them.

Reply:
ok waiting for this then!

Reply:
I think we'll do this in the upcoming propagation because we have at least 2 other changes to propagate.